src/app.py

```diff
@@ -20,9 +20,12 @@
 def get_prompt():
 	""" Build a prompt based on the current logged in user or guest """
 	username = 'guest'
+	working = ''
 	if current_user.is_authenticated:
 		username = current_user.username
-	return f'{username}@{request.host} $ '
+		#how to call the click command for working here instead of writing the query again?
+		working = models.Directory.get(models.Directory.id == current_user.working_directory).get_full_path()
+	return f'{username}@{request.host} $ ' + working + '> '
```

Comment:
prompt emulates the default Mac terminal prompt

```python
return f'{username}@{request.host}:{working} $ '
```

Reply:
Fixed.